store/store_test.go

```diff
@@ -2622,6 +2622,34 @@ var MockUpdatesWithDeltasJSON = `
 }
 `
 
+func (t *remoteRepoTestSuite) TestUbuntuStoreRepositoryDefaultsNoDeltasOnCore(c *C) {
+	restore := release.MockOnClassic(false)
```

Comment:
I would prefer if we would test both cases (classic and non-clasic) here. Maybe something like:
```
for _, t := range []struct{
 onClassic bool
 deltaFormatStr string
} {
  {false, ""},
  {true, "xdelta3"},
{
  .... slightly tweaked body of the current code
}
```